nullaway/src/main/java/com/uber/nullaway/handlers/LibraryModelsHandler.java

```diff
@@ -614,6 +614,11 @@ private static class DefaultLibraryModels implements LibraryModels {
         new ImmutableSetMultimap.Builder<MethodRef, Integer>()
             .put(methodRef("java.util.Optional", "orElse(T)"), 0)
             .put(methodRef("com.google.common.io.Closer", "<C>register(C)"), 0)
+            .put(methodRef("java.util.Map", "getOrDefault(java.lang.Object,V)"), 1)
+            .put(
+                methodRef(
+                    "com.google.common.collect.ImmutableMap", "getOrDefault(java.lang.Object,V)"),
+                1)
```

Comment:
Ideally this would be inherited always and we wouldn't need to list it explicitly here.  Unfortunately, for the combination of an inherited model and `AcknowledgeRestrictiveAnnotations` this does not seem to happen.  Rather than trying to fix that subtle corner case, I decided to just list the method explicitly.

Reply:
I think, in retrospect, that it makes some amount of sense that the resolution order is (for `B extends A`):

- Explicit library models for overriding method `B.foo(Object o)`
- Restrictive annotations in the jar for overriding method `B.foo(Object o)`
- Explicit library models for overridden method `A.foo(Object o)`
- Restrictive annotations in the jar for overridden method `A.foo(Object o)`

We just want to make sure that library models for a given method override near everything else for that exact method.

That said, this case is still weird, because I doubt `ImmutableMap` has `@Contract(...)` and we are already assuming all arguments and the return are `@Nullable` in the more general case, no? I guess my model is a bit flawed here in that adding a library model of `NULL_IMPLIES_NULL_PARAMETERS` is not _actually_ the same as annotating with `@Contract(...)`. For example, I would have expected us to also need to assert that `Map::get` can return `@Nullable` explicitly in a different library model, but adding the method here actually _implies_ that, whereas adding an `@Contract` wouldn't have. 

Either way, happy with this explicit fix for now.

Reply:
> That said, this case is still weird, because I doubt `ImmutableMap` has `@Contract(...)` and we are already assuming all arguments and the return are `@Nullable` in the more general case, no? I guess my model is a bit flawed here in that adding a library model of `NULL_IMPLIES_NULL_PARAMETERS` is not _actually_ the same as annotating with `@Contract(...)`. For example, I would have expected us to also need to assert that `Map::get` can return `@Nullable` explicitly in a different library model, but adding the method here actually _implies_ that, whereas adding an `@Contract` wouldn't have.

I think the issue here was that the documentation for `com.uber.nullaway.LibraryModels#nullImpliesNullParameters` was unclear.  I tried to clarify here: https://github.com/uber/NullAway/pull/724/files#diff-a42b334be0392157034a866655e5e0596680448baad4ed205d09858d061b3a95R87 Does that help?

Reply:
Appreciate the updated documentation! But, in the spirit of full transparency, I hadn't RTFM, I was just going with what I remember `nullImpliesNullParameters` meaning, that my mistaken recollections matched our mistaken docs is purely coincidental 😅